### DISCUSSION TITLE
Setup and running without any problems...

### DISCUSSION THREAD

**Discussion by bcardi0427** [MatthewBCooke/BirdNET-Pi#5] [General]
My Birdnetpi on LePotato had a problem and when I restarted it from a fresh install it kept stopping about every half hour, Then I saw this fork and decided to give it a try. The only thing I did different is I used Turnkey's 17.1 Debian Template as I've had really good results running it. Everything loaded fine and the site is up and running with no problems for the last hour or so. It is available through my cloudflare proxy at https://birdnetpi.bcardi.com

Thanks for all your work, now that I have it in proxmox it will be very easy to back it up.

**Comment by kspearrin** [comment]
Same. Running it on proxmox without any issues,  using Ubuntu 22. I had to change the Caddyfile to reference PHP 8 instead of 7, but that was all.

Any reason why you wouldn't PR this back to the upstream repo?
